### DISCUSSION TITLE
Works great! Except when booting?

### DISCUSSION THREAD

**Discussion by icsy7867** [tiny-pilot/tinypilot#1224] [General]
Testing using a RHEL9 bootable USB drive.  Everything works great, except I dont see the display, just a grey screen.  Mouse and keyboard work.  Any ideas?

Other things work fine, I.E if I am fully booted into my Mac, works as intended.  Using the latest. Just installed yesterday.

**Comment by mtlynch** [comment]
Thanks for reporting this!

Unfortunately, some systems (especially laptops and all-in-ones) refuse to display to an external display during BIOS/boot. What's the make/model of the RHEL9 system? And do you see a gray screen or a message that says ["No Signal"](https://tinypilotkvm.com/faq/no-signal)?

**Reply by icsy7867** [reply]
It's a dell Latitude 7420.  I do not see a "No Signal".  

When the installer is booting up, I see the verbose messages/lines.  But as soon as the GUI loads it's just an off gray screen (Note, it's not "Off", but gray.  I wonder if the RHEL installer has enough video drivers/software that it is treating the display as a second monitor.

**Reply by mtlynch** [reply]
Oh, it sounds like RHEL actually is displaying to the TinyPilot, but in "extend" mode rather than "mirror" mode. Is it possible for you to drag the window from the display in the laptop's monitor left or right over to the TinyPilot's display?

**Comment by icsy7867** [comment]
That's exactly what's happening.  I can't move screens, but the mouse moves to the grey screen.

I have an inkling that there is a grub2 setting that I can force, to force/default to a screen mirror.  However I havent found what I want yet with grub.

However, at least with newer dells, there is a software button to switch monitors that works in POST which is neat. 
FN + F8

![image](https://user-images.githubusercontent.com/3788162/206732036-56380b14-8ac9-4fba-9a80-d0fe7a0a6e9a.png)

Works great for bios settings, boot options, grub.  Still defaults to the other screen after booting to the media.  I will play around with it.
